habitat-lab/habitat/datasets/rearrange/rearrange_dataset.py

```diff
@@ -78,3 +95,65 @@ def from_json(
             rearrangement_episode.episode_id = str(i)
 
             self.episodes.append(rearrangement_episode)
+
+
+@registry.register_dataset(name="ObjectRearrangeDataset-v0")
+class ObjectRearrangeDatasetV0(PointNavDatasetV1):
```

Comment:
are these changes from the language-rearrange dataset generation code? I think its a good idea to include them, but maybe it should be a separate PR?

Reply:
Yes, these changes are from the language-rearrange dataset